src/ngrx-entity/__files__/__name@dasherize@if-flat__/__name@dasherize__.actions.ts

```diff
@@ -4,9 +4,9 @@ import { <%= classify(name) %> } from './<%= dasherize(name) %>.model';
 import { <%= classify(name) %>SearchQuery } from './<%= dasherize(name) %>.reducer';
 
 export enum <%= classify(name) %>ActionTypes {
-  Insert<%= classify(name) %> = '[<%= classify(name) %>] Insert',
-  Insert<%= classify(name) %>Success = '[<%= classify(name) %>] Insert Success',
-  Insert<%= classify(name) %>Fail = '[<%= classify(name) %>] Insert Fail',
+  Create<%= classify(name) %> = '[<%= classify(name) %>] Insert',
```

Comment:
The type string name should match the Enum name
```suggestion
  Create<%= classify(name) %> = '[<%= classify(name) %>] Create',
```